packages/api/db/docker/docker-compose.yml

```diff
@@ -26,3 +26,28 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: postgres
       POSTGRES_PORT: 5432
+  ipfs0:
```

Comment:
Can we have this in a different docker file? There are a quite of reasons to separate it:

- this is quite strange to have it in the db folder (I would suggest adding it to https://github.com/nftstorage/nft.storage/tree/main/packages/tools
- for local development testing, might be import to use a persistent local DB (previously populated with some data), instead of keeping starting and stopping one, but we still would like to run cluster container.
- we might want to debug a cluster instance

Reply:
i can move the full docker setup to a different place with a proper name definitely 

we can persist db data by just stopping the project instead of cleaning with `yarn clean`

why can't we debug cluster with this ?

Reply:
> why can't we debug cluster with this ?

Because we need to run everything in the compose. If we want to debug cluster locally, we need to run the DB docker independently, and just run our cluster locally.

Reply:
not following sorry.. if you are debug something you can just comment out the things you dont need

Reply:
We can do that, but easier to just run/change package json commands according to needs. Anyway, I think this is cleaner outside the db folder/context